robocop/config/config.py

```diff
@@ -55,15 +55,12 @@ def is_rule_enabled(self, msg):
         if self.include and not self.include_patterns:
             if msg.msg_id not in self.include and msg.name not in self.include:
                 return False
-        if self.exclude:
-            if msg.msg_id in self.include or msg.name in self.include:
+        if msg.msg_id in self.exclude or msg.name in self.exclude:
```

Comment:
You can also type 
```python
if any(x in [msg.msg_id, msg.name] for x in self.exclude):
```
but I am not sure if it is more readable.

Reply:
actually in this case self.exclude is set so it's faster to check if msg is in self.exclude that other way around. 

Your code would be then this:
```
any(m in self.exclude for m in (msg.msg_id, msg.name)
```
But I think just two ors is bit more readable than that